### ISSUE TITLE
PMM config Posters

### ISSUE THREAD

**Issue by SturmB** [mraniki/mraniki.github.io#1]
I could not find any other way of contacting you, so I'm leaving an issue on this repo in the hopes that you'll see it.

I am attempting to use [your PMM Config](https://github.com/meisnate12/Plex-Meta-Manager-Configs/tree/master/mraniki), but I cannot get the poster assets to appear. (Nor the overlays, for that matter.)

Where should I extract the `.zip` file downloaded from Mega that has all the assets, so that they will appear as the posters for the collections, as shown in your `README.md`?

**Comment by mraniki** [comment]
Your zip file (or any or your assets) should be located in your asset directory path you define in the config.
Here for me it's .../Plex-Meta-Manager/config/assets/
So for example if you want to use the poster for popular
you need to 
1) have a directory with Popular in your asset directory define in the config (in my example .../Plex-Meta-Manager/config/assets/01charts/Popular)
2) have the poster.png in the folder .../Plex-Meta-Manager/config/assets/01charts/Popular

See below the config example:
<img width="714" alt="Screenshot 2022-05-18 at 09 55 28" src="https://user-images.githubusercontent.com/8766259/168987501-8fdbd7a4-833a-4948-a072-f3f7e19163e7.png">

and a sample for assets directory structure:
<img width="254" alt="Screenshot 2022-05-18 at 09 57 58" src="https://user-images.githubusercontent.com/8766259/168987964-9961a971-571d-4b3d-ace4-5ed1364d6776.png">


Let me know if that help.

**Comment by SturmB** [comment]
Since `asset_folders` defaults to `true` and `asset_directory` is already set to `config/assets`, I was confused as to why I would need to add so many `asset_directory` entries for each library.

Then I noticed that `asset_depth` defaults to `0`. I increased it to `3` and now PMM picks up the posters! I guess the scanner just wasn't looking deep enough and stopped at `config/assets`. So I didn't need to add all those `asset_directory` entries for each library after all, yay!

Now to figure out the overlays…
